### DISCUSSION TITLE
audio bitrate

### DISCUSSION THREAD

**Discussion by rolfausohnder** [aclap-dev/video-downloadhelper#949] [Help and Q&A]
### Your question

![Screenshot 2024-06-29 160034](https://github.com/aclap-dev/video-downloadhelper/assets/71959319/8b5f5435-0292-4fb6-a153-87673ba6ca32)

Some beginner's questions: 
can I see the bitrate of the audio stream when selecting quality and format ?
does VDH deliver the audio stream as it would be delivered if I were to watch the video direcly on youtube ?
or does some decoding/encoding take place ?
Sorry if that has been asked already, didn't find it so far.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 5,
    "successfull_dl": 4
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

VDH does not display the audio properties, only the video properties.

As far as I know, what VDH downloads is the same as what you would see & hear if you played the item in the player on the YouTube web page.  This is true on any site, not just YouTube.  However, YouTube mercilessly compresses most of their content so the quality is probably not ideal.  Even at high resolutions, the video bit rates are rather low.  Audio bit rates are usually about 128Kbps, which is sort of middle of the road.  After you download something, just pop open the file's properties & tab over to the Details tab.  That will show you what you got.

**Reply by rolfausohnder** [reply]
Thank you ! I'm only interested in the audio and did some tests, as you said I got 128 Kbps. They contained frequencies up to 15 kHz,. so they had not been upsampled stream of lower bitrates.
